morpheus/_lib/include/morpheus/messages/control.hpp

```diff
@@ -68,6 +68,16 @@ class ControlMessage
      */
     void add_task(const std::string& task_type, const nlohmann::json& task);
 
+    /**
+     * @brief Get the tasks for the control message.
+     */
+    const nlohmann::json& tasks();
+
+    /**
+     * @brief Get the metadata for the control message.
+     */
+    const nlohmann::json& metadata();
```

Comment:
```suggestion
    const nlohmann::json& metadata() const;
```
Should be `const`, right? Dont want to allow editing this directly outside of the `add`/`remove` functions